setup.py

```diff
@@ -38,10 +38,15 @@
         'Environment :: Web Environment',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Zope Public License',
+        'Programming Language :: Python',
```

Comment:
@mgedmin You are right. I was believing this would be necessary to find the package on PyPI using the "Browse packages" function. But I was wrong: Searching for `Programming Language :: Python` lists `zope.testbrowser` even without having this concrete classifier.
See https://pypi.python.org/pypi?:action=browse&show=all&c=214